src/data-workspace/category-combo-table/calculate-totals.js

```diff
@@ -0,0 +1,14 @@
+export const calculateColumnTotals = (matrix) =>
+    matrix.length < 2
+        ? matrix[0].map((v) => Number(v) || 0)
+        : matrix.reduce((totals, currRow) =>
+              totals.map((currTotal, i) => sum(currTotal, currRow[i]))
+          )
```

Comment:
Nop, again, note that when `reduce` does not have a default value, the first element in the array is the `accumulator`, the `second` will be the `current`-element. So `reduce` will through in case of `matrix.length === 1`.